ec-2_windows2019.tf

```diff
@@ -0,0 +1,23 @@
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+      version = "3.69.0"
```

Comment:
we can use the latest version which is 4.29
https://github.com/hashicorp/terraform-provider-aws/releases/tag/v4.29.0